lua/bufferline/layout.lua

```diff
@@ -105,7 +98,7 @@ local function calculate(state)
 
   local buffers_width = available_width - tabpages_width
 
-  local buffers_length               = len(state.buffers)
```

Comment:
I removed `utils.len` since there are two alternatives for it:

1. Call `table.getn(foo)` for tables, `string.len(foo)` for strings, etc. This still incurs the cost of calling a function though.
2. Use `#foo` directly, which is a bit faster.